### DISCUSSION TITLE
Anyway we can delete the clips?

### DISCUSSION THREAD

**Discussion by goulaobangzi** [blakeblackshear/frigate#805] [Ask A Question]
Is there any way we can delete clips from the web API? or maybe from the db?

**Comment by paularmstrong** [comment]
Not yet. They are currently auto-deleted according the the [retention setting](https://blakeblackshear.github.io/frigate/configuration/index#clips) in the config

**Comment by KonTiki1957** [comment]
@paularmstrong
Hi to everyone, I'm experimenting with frigate in HA since a couple of days. One of my problems is exactly this: as a test I configured the retention settings to just one day, but it doesn't seem to be working, I already have 6 days of snapshots and clips. Any advise?

**Reply by paularmstrong** [reply]
You can stop the docker container, delete clips and the sqlite DB from your filesystem, then restart the container.

**Reply by KonTiki1957** [reply]
Ok thanks, am I to assume it's a bug or I messed up with something?

**Reply by blakeblackshear** [reply]
If your retention settings are not working, please open an actual issue and provide your config.

**Reply by KonTiki1957** [reply]
ok thanks :-)
